schema.json

```diff
@@ -0,0 +1,37 @@
+
+{
+    "database_name" : "Check-in",
+
+    "Tables" : [
```

Comment:
I don't remember agreeing on this and the link does not load.
I know that there was an example in one of the emails about the schema file and it was text file, but this does not imply that we must implement it this way.
Anyway, I think JSON file will make it easier for indexing rather than reading every word in a text file just to get the database name.

Reply:
This is a text file. 
+1 for Mariam 

@josephShokry  we need to discuss this

Reply:
@bangkook  could you please resolve the comments that are done ?